topics/admin/tutorials/interactive-tools/tutorial.md

```diff
@@ -893,8 +894,186 @@ Once the playbook run is complete and your Galaxy server has restarted, run the
+>
+> <br>
+> 
+> {:start="6"}
+> 6. Run the following commands to create to create a function that sends a NOTIFY message to the channel `gxitproxy` and a trigger that runs the function every time the table `gxitproxy` changes.
+>
+>    > <code-in-title>SQL</code-in-title>
+>    > ```sql
+>    > CREATE OR REPLACE FUNCTION notify_gxitproxy()
+>    > RETURNS trigger AS $$
+>    > BEGIN
+>    >   PERFORM pg_notify('gxitproxy', 'Table "gxitproxy" changed');
+>    >   RETURN NEW;
+>    > END;
+>    > $$ LANGUAGE plpgsql;
+>    >
+>    > CREATE TRIGGER gxitproxy_notify
+>    > AFTER INSERT OR UPDATE OR DELETE ON gxitproxy
+>    > FOR EACH ROW EXECUTE FUNCTION notify_gxitproxy();
+>    > ```
+>    > {: data-cmd="true"}
+>    {: .code-in}
+>
+>    > <code-out-title>SQL</code-out-title>
+>    > ```
+>    > CREATE FUNCTION
+>    > CREATE TRIGGER
+>    > ```
+>    {: .code-out}
+>
+{: .hands_on}
+
+The next step is configuring Galaxy and the Interactive Tool Proxy to use the new database.
+
+> <hands-on-title>Configure Galaxy and the Interactive Tool Proxy</hands-on-title>
+>
+> 1. Adjust your `group_vars/galaxyservers.yml` file as follows.
+>
+>    {% raw %}
+>    ```yaml
+>    # ... existing configuration options ... #
+>
+>    galaxy_config:
+>      galaxy:
+>        # ... existing configuration options in the `galaxy` section ...
+>        # interactivetools_map: "{{ gie_proxy_sessions_path }}"  # comment, remove or leave this line in place (it will be overridden by the option below)
+>        interactivetools_map_sqlalchemy: "{{ gie_proxy_sessions_path }}"
+>        # ... other existing configuration options in the `galaxy` section ...
+>
+>    # ... other existing configurations ... #
+>
+>    gie_proxy_sessions_path: "postgresql:///gxitproxy?host=/var/run/postgresql"
+>    ```
+>    {% endraw %}
+>
+> 2. Run the playbook:
+>
+>    ```
+>    ansible-playbook galaxy.yml
+>    ```
+>
+{: .hands_on}
+
+That's it, once the playbook run is complete, both Galaxy and the Interactive Tools Proxy will be storing the Interactive Tools Session Map in PostgreSQL.
 
+# Final Notes
+As mentioned at the beginning of this tutorial, Galaxy Interactive Tools are a relatively new and rapidly evolving feature. At the time of writing, there is no official documentation for Interactive Tools. Please watch the [Galaxy Release Notes][galaxy-release-notes] for updates, changes, new documentation, and bug fixes.
```

Comment:
https://docs.galaxyproject.org/en/master/admin/special_topics/interactivetools.html is the documentation, it should be referenced.

Reply:
In this PR I am not touching anything that does not have to do with the HA setup. I proposed this sort of useful changes in #5178. That one takes care of [referencing it](https://github.com/galaxyproject/training-material/pull/5178/files#diff-2b9d16350f091eba9b5dddf85c5908f5554f62ddd1989309280ca2d4a403ec34R62) ([final notes were moved](https://github.com/galaxyproject/training-material/pull/5178/files#diff-2b9d16350f091eba9b5dddf85c5908f5554f62ddd1989309280ca2d4a403ec34L895-L900) to the [comment at the beginning](https://github.com/galaxyproject/training-material/pull/5178/files#diff-2b9d16350f091eba9b5dddf85c5908f5554f62ddd1989309280ca2d4a403ec34L54-R66))

Reply:
Ok. In the future, if you're making separate changes to one file in the GTN, please just make one PR for the entire set of changes to a file, it'll be easier for us to review and ensure the final result is coherent.